backend/coins/btc/handlers/handlers.go

```diff
@@ -121,15 +121,27 @@ func (handlers *Handlers) formatAmountAsJSON(amount coin.Amount, isFee bool) For
 
 func (handlers *Handlers) formatAmountAtTimeAsJSON(amount coin.Amount, timeStamp *time.Time) FormattedAmount {
 	accountCoin := handlers.account.Coin()
-	conversions := map[string]string{}
-	estimated := false
-	if timeStamp != nil {
+	rateUpdater := handlers.account.Config().RateUpdater
+	formatBtcAsSat := util.FormatBtcAsSat(handlers.account.Config().BtcCurrencyUnit)
+	var conversions map[string]string
+	var estimated bool
+
+	if timeStamp == nil {
+		conversions = coin.Conversions(
```

Comment:
Even if I suggested this approach, I'm still not sure if it wouldn't be better to check for the nil timestamp and possibly fallback on `coin.Conversions` directly inside `coin.ConversionsAtTime`.
No strong opinion, in any case. If you like this let's keep it.

Reply:
I think the current way makes it more explicit so I'm fine with it, but also I don't feel strongly about it :)